### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: docs/docs/constraints.md [hunk 1]
```diff
@@ -4,19 +4,19 @@ title: Constraints
 
 # Constraints
 
-Constraints control where apps run to allow optimizing for fault tolerance or locality. They are made up of three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
+Constraints control where apps run to allow optimizing for either fault tolerance (by spreading a task out on multiple nodes) or locality (by running all of an applications tasks on the same node). Constraints have three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
```
Comment (gkleiman) [inline docs/docs/constraints.md]: nit: should we replace slave with agent?

**Comment by gkleiman** [comment]
@sascala thanks, looks much better! Just a small comment.

File: docs/docs/constraints.md (hunk 1 above)
Comment (sascala) [inline docs/docs/constraints.md]: yes

Comment (sascala) [inline docs/docs/constraints.md]: @gkleiman Done.

**Comment by gkleiman** [comment]
@sascala thanks!

**Pull request merged**